index.js

```diff
@@ -392,6 +413,16 @@ class MillerColumnsElement extends HTMLElement {
     }
   }
 
+  /** Focus the miller columns item asociated with a topic */
+  focusTopic(topic: ?Topic) {
+    if (topic instanceof Topic && topic.checkbox) {
+      const item = topic.checkbox.closest(`.${this.classNames.item}`)
+      if (item instanceof HTMLElement) {
```

Comment:
This could be simplified to `if (item)`, I think - I don't see what value it would be other than HTMLElement or null.

Reply:
Flow will error on this as it may return _undefined_ – depending on the markup quality/consistency.

Reply:
Interesting - I wouldn't see that as an issue, as it's a falsey value, but understand it's being enforced for the greater good.